jest.config.mjs

```diff
@@ -0,0 +1,23 @@
+import nextJest from 'next/jest.js'
```

Comment:
I've never used this and didn't know next and jest are so closely compatible. This is great. I wonder if the `.mjs` could be converted to `.ts`. Can you try it? Otherwise, keep it. It's also not a problem but we can chat about esm and commonjs at a later time.